Add model registry ui network policy

ederign opened the pull request:
## ✏️ Summary of Changes
This PR adds model registry ui network policy. It's a blocker to include Model Registry UI.

## 📦 Dependencies
No dependencies

## 🐛 Related Issues
No dependencies

## ✅ Contributor Checklist
  - [X] I have tested these changes with kustomize. See [Installation Prerequisites](https://github.com/kubeflow/manifests#prerequisites).
  - [X] All commits are [_signed-off_](https://github.com/kubeflow/community/tree/master/dco-signoff-hook#signing-off-commits) to satisfy the DCO check.

---     
 
> You can join the CNCF Slack and access our meetings at the [Kubeflow Community](https://www.kubeflow.org/docs/about/community/) website. Our channel on the CNCF Slack is here [**#kubeflow-platform**](https://app.slack.com/client/T08PSQ7BQ/C073W572LA2).

ederign commented:
@Griffin-Sullivan thank you so much for the pointers!

ederign commented:
@juliusvonkohout, please let me know if I need to send a PR to cherry-pick this to 1.10 branch1.

Also, should I update manifest docs to remind people to create this network policy?

ederign commented:
/cc @lucferbux

ederign commented:
I see a failure in YAML action, but it seems correct. (it's a copy and paste for others).
 
 For the record, other policies are failing as well:
 
 ```
 yamllint /Users/ederign/src/kubeflow/manifests/common/networkpolicies/base/katib-ui.yaml                                                          (kind-kubeflow/default)
/Users/ederign/src/kubeflow/manifests/common/networkpolicies/base/katib-ui.yaml
  9:7       error    wrong indentation: expected 4 but found 6  (indentation)
  12:11     error    wrong indentation: expected 8 but found 10  (indentation)
  14:5      error    wrong indentation: expected 2 but found 4  (indentation)
  15:9      error    wrong indentation: expected 6 but found 8  (indentation)
  17:15     error    wrong indentation: expected 12 but found 14  (indentation)
  20:19     error    wrong indentation: expected 16 but found 18  (indentation)
  22:5      error    wrong indentation: expected 2 but found 4  (indentation)
```

juliusvonkohout commented:
> I see a failure in YAML lint action, but it seems correct. (it's a copy and paste for others).
> 
> For the record, other policies are failing as well:
> 
> ```
> yamllint /Users/ederign/src/kubeflow/manifests/common/networkpolicies/base/katib-ui.yaml                                                          (kind-kubeflow/default)
> /Users/ederign/src/kubeflow/manifests/common/networkpolicies/base/katib-ui.yaml
>  9:7       error    wrong indentation: expected 4 but found 6  (indentation)
>  12:11     error    wrong indentation: expected 8 but found 10  (indentation)
>  14:5      error    wrong indentation: expected 2 but found 4  (indentation)
>  15:9      error    wrong indentation: expected 6 but found 8  (indentation)
>  17:15     error    wrong indentation: expected 12 but found 14  (indentation)
>  20:19     error    wrong indentation: expected 16 but found 18  (indentation)
>  22:5      error    wrong indentation: expected 2 but found 4  (indentation)
> ```

Then please check the indentation again, you seem to have too many spaces before the dash.

juliusvonkohout commented:
> @juliusvonkohout, please let me know if I need to send a PR to cherry-pick this to 1.10 branch1.

It all goes into the master and I synchronize from time to time from master into 1.10, so nothing you need to do.

> Also, should I update manifest docs to remind people to create this network policy?

No, it shall be there by default. Just please add it to the networkpolicies kustomization.yaml

ederign commented:
@juliusvonkohout I've fixed the indentation and added to kustomize: https://github.com/kubeflow/manifests/blob/e33621851e1fe43c806cbd29752702d33bcbbca2/common/networkpolicies/base/kustomization.yaml#L20

tarilabs reviewed (commented):
/lgtm

juliusvonkohout commented:
/lgtm
/approve
